<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditformUser</class>
 <widget class="QDialog" name="EditformUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="nomLabel_2">
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dateDInscriptionLabel_2">
      <property name="text">
       <string>Date d'inscription</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emailLabel_2">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="emailLineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="sexeLabel_2">
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="sexeLineEdit_2"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomLineEdit_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="dateDInscriptionLineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier un membre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
